int8 redCaseValue
string solvedSudocube
int8 redCasePosition
---
